comet/api/endpoints/stream.py

```
@@ -24,6 +27,38 @@
 streams = APIRouter()
 
 
+def _build_stream_response(
+    request: Request,
+    content: dict,
+    is_public: bool = False,
+    vary_headers: list = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)